python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -1883,6 +1883,8 @@ def __init__(self):
     def callback(self, pre, post, node_map):
         act = node_map[self.act][0]
         axis = post.attrs.axis
+        if any([isinstance(dim, tvm.tir.expr.Any) for dim in pre.checked_type.shape]):
```

Comment:
Can you give some context in PR why the change?

Reply:
@dgolubovicTT , this change fixes [#642](https://github.com/tenstorrent/tt-forge-fe/issues/642), the current status of this model is mentioned in the description [here](https://github.com/tenstorrent/tt-forge-fe/pull/865).

Reply:
Yes, in that case we skip removal of squeeze containing dynamic shapes, warning can be added. Squeeze op is supported in mlir as mentioned [here](https://github.com/tenstorrent/tt-forge-fe/blob/8ed75136aba734c23300cd92a73e71192dbbcf46/forge/csrc/passes/lower_to_mlir.cpp#L587). 
Densenet model is failing with unsupported ops - argwhere and scatter_elements after this change

Reply:
Ok, can you add a single squeeze test that will test this case, just to be sure everything after tvm will work.

Reply:
[These](https://github.com/tenstorrent/tt-forge-fe/blob/79e186f17dd5aa0e14a3a358a66de47f503c13ed/forge/test/mlir/test_ops.py#L981) test cases for squeeze op were added as a part of this [PR](https://github.com/tenstorrent/tt-forge-fe/pull/528/files#diff-9094f8f34c15528e4fad062984363b21b8e2fdd4b19c6d49878c16cebea33484) when DecomposeMultiDimSqueeze callback was added.

Reply:
But is any of them actually testing this code path where this condition is not satisfied:
```
 if any([isinstance(dim, tvm.tir.expr.Any) for dim in pre.checked_type.shape]):
```

Reply:
I will try this and let you know.